src/frontend/test_runner/tests/testdata/order_by.yaml

```diff
@@ -43,3 +47,27 @@
     create table t (v1 bigint, v2 double precision);
     select * from t order by v;
   binder_error: 'Item not found: output column "v"'
+- sql: |
+    create table t (v1 bigint, v2 double precision);
+    select * from t order by v1 desc limit 5;
+  batch_plan: |
+    BatchTopN { order: [$0 DESC] limit: 5, offset: 0 }
+      BatchExchange { order: [], dist: Single }
+        BatchScan { table: t, columns: [v1, v2] }
+  stream_plan: |
+    StreamMaterialize { columns: [v1, v2, _row_id#0(hidden)], pk_columns: [v1] }
```

Comment:
Why we only have `v1` in pk_columns? If there are two elements of the same value from topN, we should be able to distinguish them.

Reply:
Yes, as I had commented:

> The PK derivation of TopN needs to be fixed in later PRs.